gamemodes/terrortown/gamemode/shared/hud_elements/tttvoice/pure_skin_voice.lua

```diff
@@ -0,0 +1,168 @@
+--- @ignore
+
+local base = "pure_skin_element"
+
+DEFINE_BASECLASS(base)
+
+HUDELEMENT.Base = base
+
+if CLIENT then
+    local padding = 6
+
+    local baseDefaults = {
+        basepos = { x = 0, y = 0 },
+        size = { w = 240, h = 42 },
+        minsize = { w = 120, h = 42 },
+    }
+
+    function HUDELEMENT:Initialize()
+        self.scale = 1.0
+        self.basecolor = self:GetHUDBasecolor()
```

Comment:
I think this is unused in the whole component

Reply:
i think it still is unused

Reply:
![image](https://github.com/TTT-2/TTT2/assets/13639408/4ae2c64f-fb14-459d-a5cd-84e86969caeb)

hence I removed it

Reply:
It is still set in the layout function

Reply:
oh, I will take care of it